release/_package.sh

```diff
@@ -154,7 +171,7 @@ function package_release {
 
 	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
 
-	cp -a sql liferay-${LIFERAY_RELEASE_PRODUCT_NAME}-sql
+	cp -a sql liferay-"${LIFERAY_RELEASE_PRODUCT_NAME}"-sql
```

Comment:
@pgydev, can you add this to a separated commit?

The commits should be atomic, e.g. ideally they should do a single action. :slightly_smiling_face: